packages/eslint-plugin/src/configs/recommended.json

```diff
@@ -192,7 +201,7 @@
             "rxjs/Rx"
         ],
         "no-throw-literal": "error",
-        "no-trailing-spaces": "error",
```

Comment:
I know typescript-eslint has a [pretty strong stance](https://typescript-eslint.io/linting/troubleshooting/formatting/) against using ESLint as a formatter, and I tend to agree with that philosophy.  I kind of wonder if it might make sense to have "formatter" presets as a separate config rather than part of the default recommended, though Prettier and other formatters typically provide their own configuration that will turn conflicting ESLint rules off, so maybe it's not an issue to include them by default.

Regardless, I think this is still fine for now, but maybe something to consider.